doc/nrf/known_issues.rst

```diff
@@ -1230,6 +1230,18 @@ NCSDK-8372: Project name collision causes SES Nordic Edition to load the wrong p
   **Workaround:** If the path in :guilabel:`Build Directory` points to the wrong project, select the correct project by using the :guilabel:`...` button for :guilabel:`Projects` and navigating to the correct project location.
   The build directory will update automatically.
 
+Toolchain
+*********
+
+.. rst-class:: v1-5-0 
```

Comment:
Please verify the NCS versions to which this applies. (Maybe 1.5.1? It's too early to mention 1.6 anyway)

Reply:
as this is a toolchain issue when building TF-M, then using that toolchain will affect all releases that includes TF-M sample, which were introduced with 1.5.

Added 1.5.1 in addition.